packages/components/src/components/date-picker/date-picker-native.jsx

```diff
@@ -51,7 +51,7 @@ const Native = ({
                 {value && (
                     <Text
                         size='xs'
-                        color='prominent'
+                        color='var(--text-color)'
```

Comment:
Will this be acceptable for the `color` prop?
Because the style `props` in `class_styles` has the `var(--text-` as a prefix already.

Reply:
Ah yeah, I think it just worked because it was incorrect 😆 I changed it to a style prop